README.md

```diff
@@ -0,0 +1,228 @@
+- [About](#about)
+  * [Install](#install)
+  * [Example](#example)
+    + [Hello world with default AMQP library](#hello-world-with-default-amqp-library)
+    + [Same hello world with go-mq](#same-hello-world-with-go-mq)
+  * [Config in depth](#config-in-depth)
+  * [Error handling](#error-handling)
+  * [Epilogue](#epilogue)
+
+# About
+
+This package provides an ability to incapsulate creation and configuration [AMQP](https://www.amqp.org) entities 
```

Comment:
This package provides an ability to incapsulate creation and configuration [AMQP](https://www.amqp.org) entities 
like queues, exchanges, producers and consumers in declarative way just with one config

This package provides an ability to encapsulate creation and configuration [AMQP](https://www.amqp.org) of  queues, exchanges, producers, consumers and similar entities in a declarative way with a single config.